recipes/protobuf/all/conanfile.py

```diff
@@ -79,30 +129,73 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
Would it be possible to also set the names for the `pkg_config` generator?

Reply:
Good catch.
I would suggest that we split that out in a new issue to not stale this any longer

Reply:
https://github.com/conan-io/conan-center-index/issues/3493